README.md

```diff
@@ -137,17 +155,15 @@ Instantiate a new `Storj` object used to communicate with the storj network. The
 
 If you need to use authentication in your application, we strongly suggest you use the `key` method as it provides a higher level of security.
 
-Both `basicAuth` and `key` are optional, but you may only provide one or the other. If you use `basicAuth`, the library will attempt to create a public/private key pair for you and register it to your account. If you provide a `key`, this key will be used to authenticate every request moving forward.
+Both `basicAuth` and `key` are optional, but you may only provide one or the other. If you use `basicAuth`, the library will assume that you already have registered a public key with the bridge you are authenticating with. To create a public/private key pair for you and register it to your account you can use the `getKeypair` and `registerKeypair` API. If you provide a `key`, this key will be used to authenticate every request moving forward.
```

Comment:
`library will assume` how does it make this assumption?

What is the behaviour of the library if you attempt to make a call using `basicAuth` that requries a `KepPair`, i.e. uploading a file? Will it return an error asking for you to generate a key, will it generate one for you, etc?

Would it be best to recommend `basicAuth` only be used to immediately generate a `KeyPair`, and then the library will use `KeyPair` for all requests from that point forward?